pyproject.toml

```diff
@@ -41,17 +41,17 @@ dependencies = [
 # These optional dependencies are for building the
 # application as a part of a Github Action.
 ci = [
-    'pyinstaller==4.5.1',
+    'pyinstaller>=4.5.1',
```

Comment:
pyinstaller tends to be very finicky. We should probably try the build with the most recent version and pin the requirements to it, rather than do blanket `>=`

Reply:
Now pinned to minor version